schemas/package.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: package
+  title: Package
+  group: 2
+  short: These fields contain information about an installed software package.
+  description: >
+    These fields contain information about an installed software package. It contains general information about a
+    package, such as name, version or size. It also contains installation details, such as time, URL or
+    user who installed the package.
```

Comment:
Do you have a way to determine which user installed the package, after the fact? If so, it may be worth making `user` reusable under package (add `- package` [in this list](https://github.com/elastic/ecs/blob/v1.1.0/schemas/user.yml#L15-L20)) :-)

If you're unsure, that's also fine. We can take out the mention in the description for now, and add it later if needed. It's much easier to continue adding than to take out or change after it's in ;-)

Reply:
Never mind. Just realized this was already in.

It's Friday night, time to log off 😂